apps/src/templates/studioHomepages/CourseBlocksTools.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  regularRow: {
+    marginBottom: 20
+  }
+};
+
+const CourseBlocksTools = React.createClass({
+  propTypes: {
+    isEnglish: React.PropTypes.bool.isRequired,
+    isRtl: React.PropTypes.bool.isRequired,
+    codeOrgUrlPrefix: React.PropTypes.string.isRequired
```

Comment:
Not a big deal at all, but I've been generally importing PropTypes from React, i.e.: 
`import React, { PropTypes } from 'react';`
Such that here we can do `isRtl: PropTypes.bool.isRequired`;

This has the added benefit that in a future version of React, PropTypes is a separate library (such that React.PropTypes will be invalid and we'll need to `import PropTypes from 'prop-types';`)